src/container_registry.toit

```diff
@@ -87,6 +89,15 @@ class ContainerRegistry:
     store_
```

Comment:
Add a comment, explaining that we don't remove the entry from the revisions, so that we guarantee that a newer revision of a program still has a newer revision-number, even if it was uninstalled at some point.